app/connector/support/processor/src/test/java/io/syndesis/connector/support/processor/ErrorMapperTest.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.connector.support.processor;
+
+import static org.junit.Assert.assertThat;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.equalTo;
```

Comment:
Please use AssertJ, hamcrest doesn't create failure messages that we can use to fix the issue when a test fails. That's why hamcrest dependency is banned:
https://github.com/syndesisio/syndesis/blob/530856de8e70dae2a76b93ff9549b4e04a502bf5/app/pom.xml#L1063-L1069